src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md

```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
 prestashop-git | \n* Almost ! Starting web server now\n
 ```
 
-At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001.
+At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001. 
 
 The default credentials for the back-office are `demo@prestashop.com` / `prestashop_demo`.
 
+Default MySql credentials to connect using 3rd party programs like Sequel Pro and others:
+Username: `root`
+Password: `prestashop`
+Port: `32768`
```

Comment:
```suggestion
Port: `3306`
```
as defined here: https://github.com/PrestaShop/PrestaShop/blob/develop/docker-compose.yml#L12 :thinking:

Reply:
For me Port 3306 doesn’t work. 
I had to use ‘docker ps’ to check port. And it was written 32768->3306

Reply:
The port is maybe random :thinking:

Reply:
I confirm this port is generated dynamically, I'm not having the same one